src/univariate/continuous/normal.jl

```diff
@@ -35,6 +35,11 @@ entropy(d::Normal) = 0.5 * (log2π + 1.0) + log(d.σ)
 
 @_delegate_statsfuns Normal norm μ σ
 
+### Added by scidom: start
+logpdf(d::Normal, x::Real) = -(log(2*π)/2+log(d.σ)+abs2((x-d.μ)/d.σ)/2)
+logpdf(d::Normal, x::Number) = -(log(2*π)/2+log(d.σ)+abs2((x-d.μ)/d.σ)/2)
```

Comment:
The solution is to fix the source of the ambiguity [here](https://github.com/JuliaStats/Distributions.jl/blob/245cfe3c5546636ac2a204951d775f5b377f44e2/src/univariates.jl#L108). We need to figure out how to make this fallback work for ``Number``.